.env.example

```diff
@@ -0,0 +1 @@
+RPC_URL=https://rpc.helius.xyz/?api-key=e330d9d4-5acc-4260-a760-c80d7bab7d6c
```

Comment:
Is this protected? do not check it in if there is a quota associated with it, just use the public one